monitoring-dashboard/components/org.wso2.micro.integrator.dashboard.web/web-app/src/utils/HTTPClient.js

```diff
@@ -76,6 +78,12 @@ export default class HTTPClient {
         return this.getResource(resourcePath)
     }
 
+    static manageNode(payload) {
+        const path = '/server'
+        const basePath = 'http://localhost:9192/management'
```

Comment:
Cannot merge the PR due to this. We need to find a proper way to get the hostname/port pair.